src/app/components/grid/index.html

```diff
@@ -257,6 +257,9 @@
 					<li>
 						<span class="sky-deemphasized">Toolbar (optional)</span>
 					</li>
+					<li>
+						<span class="sky-deemphasized">Multiselect toolbar (optional)</span>
```

Comment:
When I serve locally, I don't see either of these changes to the grid topic. Do you see them in place when you serve locally? I saw the changes to the flyout topic, so I don't know why I wouldn't see these.

Reply:
Nevermind. I figured out what the problem is....